glide.yaml

```diff
@@ -18,23 +18,23 @@ import:
 - package: github.com/spf13/viper
   version: v1.0.0
 - package: github.com/tendermint/abci
-  version: v0.9.0
+  version: sdk2
```

Comment:
Am assuming these will be updated later on?

Reply:
yes